cmd/podman/tag.go

```diff
@@ -29,15 +29,20 @@ var (
 
 func init() {
 	tagCommand.Command = _tagCommand
+
+	flags := tagCommand.Flags()
+	flags.BoolVarP(
+		&tagCommand.Undo, "undo", "u", false,
+		"Undo the latest tag operation and restore the previous tag",
```

Comment:
Why only the latest tag?  How is this different the rm of a tag?
I would really like to see an explanation, example of what you are trying to do?

podman tag IMAGE foo
podman tag IMAGE bar
What do I have when I do
podman tag IMAGE --undo?

Reply:
I think we cannot remove a tag right now in podman, right? I was thinking about to add another option to be able to remove a tag from a image.

Right now it works like (only) like this:
```bash
# Downloaded alpine, which has no history initially
> ./bin/podman images --history
REPOSITORY                 TAG      IMAGE ID       CREATED       SIZE      HISTORY
docker.io/library/alpine   latest   965ea09ff2eb   7 weeks ago   5.82 MB   <none>

# Retagging alpine to `test`, now both have a history
> ./bin/podman tag docker.io/library/alpine test
> ./bin/podman images --history
REPOSITORY                 TAG      IMAGE ID       CREATED       SIZE      HISTORY
docker.io/library/alpine   latest   965ea09ff2eb   7 weeks ago   5.82 MB   docker.io/library/alpine:latest
localhost/test             latest   965ea09ff2eb   7 weeks ago   5.82 MB   docker.io/library/alpine:latest

# Now we're able to restore the previous tag of `test` by
> ./bin/podman tag localhost/test --restore
> ./bin/podman images --history
REPOSITORY                 TAG      IMAGE ID       CREATED       SIZE      HISTORY
docker.io/library/alpine   latest   965ea09ff2eb   7 weeks ago   5.82 MB   localhost/test:latest
```
We probably can also undo the tag when specifying the `docker.io/library/alpine` image, but then I would have to add more logic. Would this be intended?

Reply:
```
podman tag alpine foobar
podman tag alpine alpine:foobar
podman tag alpine docker.io/library/alpine:foobar
```
```
podman tag alpine --restore
```
I take it this just removes just docker.io/library/alpine:foobar?
What happens if I did?

podman tag foobar --restore?

I might be bikes heading here but would it be better to have the syntax be.
podman tag --remove docker.io/library/alpine:foobar
And then it just removes the tag, and cleans up the history?

If I eventually do a 
podman tag --remove docker.io/library/alpine:latest
then it would generate a 
<none> <none> image?

Reply:
Another option would be to add a 
podman untag ...